pom.xml

```diff
@@ -24,6 +25,14 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <tmc.integration.tests/>
+    <junit.version>5.5.2</junit.version>
```

Comment:
I see that we have some dependency version numbers captured here and others in the dependency itself. It also looks like we are only capturing version number up here for items that are test dependencies.

Is there a reason why we do this two different ways?  My preference would be to have everything consistent.

Reply:
I agree we should have consistency. We can have all dependency versions as properties.